### DISCUSSION TITLE
baton_cl_rows_attributes

### DISCUSSION THREAD

**Discussion by AlanL-django** [otto-torino/django-baton#285] [Q&A] [answered]
Hi. I'm currently in the process of migrating an old Suit app to Baton. Quite happy so far with tabs & menus, now I'm experimenting with changelist row attributes. I've defined a `baton_cl_rows_attributes` in my ModelAdmin, but it isn't being called when I open or refresh the ChangeList

Django 5.0.1, Baton 3.0

**Comment by abidibo** [comment]
Hi @AlanL-django my test app also runs with Django 5 and `baton_cl_rows_attributes` are working as expected.

Can you share the code?

**Reply by AlanL-django** [reply, marked as answer]
Thanks for the quick response. I had the wrong parameter names: I declared my method as 
`def baton_cl_rows_attributes(self, request, changelist):`
instead of
`def baton_cl_rows_attributes(self, request, cl):`

Working fine now with the correct parameter namss

I usually call with positional rather than named parameters in my own code because I am lazy; It looks llike you are not

**Reply by AlanL-django** [reply]
I do have one other question though - how hard would it be to make the same thing work for inlines? I mignt give it a try with `django-changelist-inline`
